protocol/x/clob/keeper/process_operations_test.go

```diff
@@ -1426,6 +1426,289 @@ func TestProcessProposerOperations(t *testing.T) {
 			},
 			expectedError: types.ErrInvalidOrderRemoval,
 		},
+		"Fails with order removal for market in final settlement": {
```

Comment:
for my understanding - didn't we add a order removal reason for this?

Reply:
we added an indexer order removal reason (for when stateful orders are forcefully cancelled), there is no ops queue OrderRemoval for this though

All stateful orders will be cancelled in EndBlocker when gov proposal is processed, so we don't need an OrderRemoval type for this in the ops queue. They will never need to be proposed